mainwindow.cpp

```diff
@@ -94,6 +99,14 @@ namespace dealii
 
 
 
+    void MainWindow::set_to_default()
+    {
+      QTreeWidgetItem * current_item = tree_widget->currentItem();
+      current_item->setText(1,current_item->text(2));
```

Comment:
With Qt, who is responsible for eventually deleting this (and all of the other objects you create)?